decidim-budgets/app/views/decidim/budgets/projects/_budget_confirm.html.erb

```diff
@@ -17,7 +17,7 @@
                 </div>
               </div>
               <div class="budget__card__list-project">
-                <span class="budget-list__data__number budget-list__number-show-for-medium">
+                <span class="budget-list__number-show-for-medium">
```

Comment:
I think this isn't doing anything
```suggestion
                <span>
```

Reply:
Or at least I can't find it in the CSS. As far as I remember, these `show-for-*` are Foundation CSS utility classes for responsiveness/accessibility.